docs/endpoints/about-endpoints.md

```diff
@@ -5,19 +5,21 @@ The endpoints module allows you to query Vortexa's data.
 The VortexaSDK currently contains the following endpoints:
 
 1. Cargo Movements
-2. Voyages
-3. Charterers
-4. Geographies
-5. Products
-6. Vessels
-7. Cargo Time Series
-8. EIA Forecasts
-9. Tonne-miles
-10. Vessel Availability
-11. Crude Onshore Inventories
-12. Freight Pricing
-13. Vessel Summary
-14. Vessel Positions
+1. Voyages
+1. Charterers
+1. Geographies
+1. Products
+1. Vessels
+1. Cargo Time Series
+1. EIA Forecasts
+1. Tonne-miles
+1. Vessel Availability
+1. Crude Onshore Inventories
+1. Freight Pricing
+1. Vessel Summary
+1. Vessel Positions
+1. Canal transit
+1. Canal transit Time Series
```

Comment:
All numbers are 1.

Reply:
<img width="628" alt="Screenshot 2024-04-09 at 10 25 20" src="https://github.com/VorTECHsa/python-sdk/assets/116007199/5ddfc541-42d0-4b31-aae2-7497dbfa4458">

Markdown is able to loop through this. `1` is easier to track if you add/remove things as you do not need to update whole list

Reply:
Today I learned!